src/settings.py

```diff
@@ -126,6 +126,7 @@ class GlobalSettings:
     stick_point_num_points_train = [1, 2]
     stick_point_num_points_test = [3, 4]
     stick_point_disable_angles = True
+    stick_point_holder_scale = 0.1
```

Comment:
is this a fraction of the stick's total width? it doesn't seem like it from the videos, since for the holder_scale=0.5 videos, it's clearly not covering half the stick

Reply:
copying from another comment bc it addresses this question but is still confusing me -- why is it the case that in the videos, the holder can extend off the end of the stick? i would have thought it should be completely contained within the stick.

Reply:
it is a fraction of the stick's total width. in the holder_scale=0.5 video, the size of the holder is half the size of the stick.

I made it possible for the holder to extend off the stick because I wanted it to be common that the stick cannot just be grasped from the bottom. if I made it such that the holder was always somewhere contained in the stick's length, and I randomized that relative location, then it would be too often possible to grasp the stick from the end.

Reply:
ah okay that makes a lot of sense